src/Highlight.ts

```diff
@@ -17,11 +17,13 @@ export default class Highlight {
       this._id = (new Date()).getTime().toString();
       this._range = arg;
       this._content = arg2;
-    } else {
+    } else if (content) {
       this._id = arg;
       this._range = arg2;
       this._content = content;
     }
+
+    throw new Error('invalid constructor arguments');
```

Comment:
Doesn't this cause the constructor to always throw an Error? Maybe I missed something by just viewing the diff.